docs/mod-lists/general-mod-list.md

```diff
@@ -42,10 +42,28 @@ Select your Minecraft version below to view compatible mods.
     
     - Download: https://sk1er.club/mods/text_overflow_scroll
     - Support: https://discord.gg/sk1er
+
+    ### Chatting
+    Adds extremely customizable utilities such as chat tabs, screenshotting messages, and message copying
     
-    ### Sk1er Old Animations
-    Adds Minecraft 1.7 animations to 1.8.9
```

Comment:
Why remove this?

Reply:
It's readded below, a bit more visible if you read the new commit changes instead of the files changed page

Reply:
My bad yeah sorry